cmd/root.go

```diff
@@ -33,11 +33,11 @@ var CfgFile string
 
 var rootCmd = &cobra.Command{
 	Use:   "apb",
-	Short: "apb is a tool to manage ServiceBundle images",
-	Long: `ServiceBundles are images that represent lifecycle components
-in that they contain all of the orchestration logic to manage
+	Short: "apb is a tool to manage Ansible Playbook Bundle images",
```

Comment:
Not really related to your PR, but it feels kind of weird to me that we have a summary of what APBs are in the CLI tool for APBs. I think people expect to see usage instructions when they run the bare command `apb`.

Reply:
Meant to add this comment onto the `Long` description line.